src/components/jogging/JoggingStore.ts

```diff
@@ -437,8 +448,8 @@ export class JoggingStore {
     this.selectedIncrementId = id
   }
 
-  setIncrementJoggingInProgress(inProgress: boolean) {
-    this.incrementJoggingInProgress = inProgress
+  setIncrementJogging(incrementJogging: IncrementalJoggingState) {
```

Comment:
ah you defined the action method but didn't use it hehe

Reply:
"I defined" :eyes: